### ISSUE TITLE
Reactively changing sort

### ISSUE THREAD

**Issue by Urigo** [matteodem/meteor-easy-search#98]
Is there a possibility to change sort without defining a new search index?

Like binding it into a dropdown to change it easily

**Comment by matteodem** [comment]
Did this answer your question?

**Comment by Urigo** [comment]
Yes thank you.
I would still love an example in the leaderboard :)

**Comment by matteodem** [comment]
Yeah sure, when I get to it I'll surely will comment into the other ticket. There are other things right now though with higher priority.

**Comment by OminStyle** [comment]
I think the documentation is still lacking although I eventually got this to work. It could be specific to my implementation but I didn't need ```var sort = EasySearch.getSearcher(this.use).defaultSort();``` In fact, it didn't work when I tried to return  ```EasySearch.getSearcher``` object.

What I did was, on the client side: ```EasySearch.changeProperty('courses','orderBy', orderBy);```
And on the server:
```
sort: function() {
    if (this.props.orderBy) {
      if (this.props.orderBy === "views") {
        return {'views': -1};
      } else {
        return {'date': 1};
      }
    }
  }
```

**Comment by matteodem** [comment]
Okay, what would be a good example for you in the docs?

**Comment by drenfr01** [comment]
Thank you for this example, I hit the exact same problem!

**Comment by drenfr01** [comment]
@matteodem First of all, thanks for creating this package, it's been very helpful! 

Did the documentation ever get updated for this (I looked and couldn't find it). I'm trying to change the sort field dynamically, e.g. on the client side a user clicks a radio button option and changes the sort from "price" to "volume". I tried the above approach using changeProperty, but it didn't seem to work...

Here's my code:

File in meteor/lib (i.e. both and server):

EasySearch.createSearchIndex('priceListIndex', {
  'collection': PriceList,
  'field': ['name'],
  'use': 'mongo-db',
  'sort': function(searchString) {
    if (this.props.orderBy) {
      if (this.props.orderBy === 'price') {
        return {'median_net_price': -1};
      } else {
        return {'volume': -1};
      }
    }
  },
  'props': {
      'orderBy': 'price'
  },
  'query': function(searchString, opts) {
    var query = EasySearch.getSearcher(this.use).defaultQuery(this, searchString);
    query.historicalData = false;
    return query;
  }
})

Client Side Update:

Template.pricelist.events({
  'click #change': function(e) {
    console.log('click');
    EasySearch.changeProperty('priceListIndex', 'orderBy', 'volume')
  }
});

Am I doing something dumb here?
